msm/src/prover.rs

```diff
@@ -417,6 +482,16 @@ where
         .map(|poly| (coefficients_form(poly), hiding(1)))
         .collect();
 
+    // @volhovm: I'm not sure we need to prove opening of fixed
```

Comment:
They should be treated as "common preprocessed input" from PlonK: https://eprint.iacr.org/2019/953.pdf, page 27.